ui/app/AppLayouts/Communities/views/PermissionsView.qml

```diff
@@ -88,50 +88,24 @@ ColumnLayout {
         ]
     }
 
-    Loader {
-        id: repeaterLoader
-        active: root.useRepeater
-        Layout.fillWidth: active
-        Layout.fillHeight: active
-
-        sourceComponent: ColumnLayout {
-            readonly property alias count: repeater.count
-
-            width: root.viewWidth
-            spacing: 24
-
-            Repeater {
-                id: repeater
-                model: root.permissionsModel
-                delegate: permissionItemComponent
-            }
-        }
-    }
+    StatusListView {
+        id: listView
+        reuseItems: true
+        model: root.permissionsModel
+        spacing: 24
+        Layout.fillWidth: true
+        Layout.fillHeight: !useFullHeight
+        height: useFullHeight ? contentHeight : 0
+        Layout.topMargin: root.topPadding
 
-    Loader {
-        id: listViewLoader
-        active: !root.useRepeater
-        Layout.fillWidth: active
-        Layout.fillHeight: active
-
-        sourceComponent: StatusListView {
-            reuseItems: true
-            model: root.permissionsModel
-            spacing: 24
-            Layout.fillWidth: true
-            Layout.fillHeight: true
-            Layout.topMargin: root.topPadding
-
-            delegate: permissionItemComponent
-        }
+        delegate: permissionItemComponent
     }
 
     Component {
         id: permissionItemComponent
 
         PermissionItem {
-            width: root.useRepeater ? 0 : root.viewWidth
-            Layout.fillWidth: root.useRepeater
+            width: root.viewWidth
```

Comment:
```qml
width: ListView.view.width
```

Reply:
I'm not sure why, when I use that, or `width: parent.width`, it ends up taking the whole width of the page, even though the ColumnLayout has `width: root.viewWidth`

![image](https://github.com/user-attachments/assets/2e14c200-6343-448b-b916-1712ea4be03e)